Scratch.R

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Is this just a scratchpad? Should it be reviewed?

Reply:
No need to review this, thanks!